topics/ui/controls/combo_box.md

```diff
@@ -23,79 +23,86 @@ comboBox.setEditable(true);
 
 ### When not to use
 
-If the number of options is finite, and there’s no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md).
+* If the number of options is finite, and there is no need to enter custom values. In this case, use a [drop-down list](drop_down.md) or [radio buttons](radio_button.md):
 
-If it’s not possible to list the most likely choices. In this case, use an [input field](input_field.md) instead.
+  ![The Theme drop-down with a predefined list of values](combobox_when_not_to_use_1.png){width="706"}
 
-![](prefill.png){width=152}
+* If it is not possible to list the most likely choices. In this case, use an [input field](input_field.md) instead:
+  
+  ![A port number input](combobox_when_not_to_use_2.png){width=706}
 
-If the list is big, and the user knows what value they need and won’t review the list. In this case, use an input field with completion.
-
-![](input_field_completion.png){width=509}
+* If the list is big, and the user knows what value they need and won’t review the list. In this case, use an input field with completion:
+  
+  ![An input field with a completion popup](combobox_when_not_to_use_3.png){width=706}
 
 ## How to use
 
-For the [label](drop_down.md#label) and the [default value](drop_down.md#default-value) apply the same rules as for the
+* For the [label](drop_down.md#label) and the [default value](drop_down.md#default-value) apply the same rules as for the
 [drop-down list](drop_down.md).
 
-If there are no values in the list initially, replace the combo box with an input field.
+* If there are no values in the list initially, replace the combo box with an input field.
 This way, users won't waste their time clicking the empty combo box to find out that there are no options available.
-
-| <format color="Red" style="bold">Incorrect</format> | <format color="Green" style="bold">Correct</format> |
-|-----------------------------------------------------|-----------------------------------------------------|
-| ![](combo_box_empty.png){width="221"}               | ![](replace_with_iput_field.png){width="216"}       |
-
-Replace the input field with a combo box after a value has been entered and confirmed.
-
-### Menu
-
-#### Control
-
-Open the combo box menu by clicking the arrow button on the right or pressing the <shortcut>Down</shortcut> arrow key when the combo box is focused.
+  
+  <table style="none" border="false">
+    <tr>
+      <td width="50%">
+        <format color="Green" style="bold">Correct</format><img src="combobox_how_to_use_1_correct.png" alt="A correct example of a repository input field without values"/>
+      </td>
+      <td width="50%">
+        <format color="Red" style="bold">Incorrect</format><img src="combobox_how_to_use_1_incorrect.png" alt="An incorrect example of a repository input field without values"/>
+      </td>
+    </tr>
+  </table>
```

Comment:
Possibly make the left example the incorrect one? So it could be viewed as before / after